assets/tests/userTestGroups.test.ts

```diff
@@ -0,0 +1,24 @@
+import appData from "../src/appData"
+import inTestGroup from "../src/userTestGroups"
+
+jest.mock("appData")
+
+describe("User test groups helper", () => {
+  // test("Assert")
+  test("Assert detect user in group", () => {
+    const group_1 = "user-test-group-1"
+    const group_2 = "user-test-group-2"
+    const mockSettings = {
+      userTestGroups: JSON.stringify([group_1, group_2]),
+    }
+    ;(appData as jest.Mock)
+      .mockImplementationOnce(() => {})
+      .mockImplementation(() => mockSettings)
+
+    // Start with no groups or keys
+    expect(inTestGroup(group_1)).toBe(false)
+    // Once groups are "loaded", check if in groups
+    expect(inTestGroup(group_1)).toBe(true)
+    expect(inTestGroup(group_2)).toBe(true)
+  })
```

Comment:
**suggestion:** it would also be nice to have a test for the case where the JSON data is malformed (as in it parses as valid JSON but isn't an array of strings).